### PR TITLE
some formatting changes

### PR DESCRIPTION
**Pull request by a14n**
## Description

fix some bad formatting (in `packages/flutter/` and `packages/flutter_tools/`)

## Related Issues

None

## Tests

None

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter_tools/lib/src/android/android_device.dart [hunk 1]
```diff
@@ -573,24 +573,22 @@ class AndroidDevice extends Device {
...
-            ...<String>['--ez', 'start-paused', 'true'],
-          if (debuggingOptions.disableServiceAuthCodes)
-            ...<String>['--ez', 'disable-service-auth-codes', 'true'],
-          if (debuggingOptions.dartFlags.isNotEmpty)
-            ...<String>['--es', 'dart-flags', debuggingOptions.dartFlags],
-          if (debuggingOptions.useTestFonts)
-            ...<String>['--ez', 'use-test-fonts', 'true'],
-          if (debuggingOptions.verboseSystemLogs)
-            ...<String>['--ez', 'verbose-logging', 'true'],
+      if (debuggingOptions.debuggingEnabled) ...<String>[
```
Comment (goderbauer) [inline packages/flutter_tools/lib/src/android/android_device.dart]: This looks strange to me. Why is the if sometimes multi-line and sometimes not?

Comment (a14n) [reply packages/flutter_tools/lib/src/android/android_device.dart]: From a formatting point of view putting `...<T>[` on the same line as `if (a)` is mainly depending on the presence of a trailing comma in the list.

```dart
// without trailing comma (v1)
if (route != null)
  ...<String>['--es', 'route', route],

// without trailing comma (v2)
if (route != null) ...<String>['--es', 'route', route],

// with trailing comma
if (route != null) ...<String>[
  '--es',
  'route',
  route,
],
```

The question to use multi-line/trailing-comma or not is more a question that depends on readability and understandability and the answer is in the responsibility of the code writer.

FWIW here's how `dartfmt` (with a line length at 120) format this code section:

```dart
    cmd = <String>[
      'shell', 'am', 'start',
      '-a', 'android.intent.action.RUN',
      '-f', '0x20000000', // FLAG_ACTIVITY_SINGLE_TOP
      '--ez', 'enable-background-compilation', 'true',
      '--ez', 'enable-dart-profiling', 'true',
      if (traceStartup) ...<String>['--ez', 'trace-startup', 'true'],
      if (route != null) ...<String>['--es', 'route', route],
      if (debuggingOptions.enableSoftwareRendering) ...<String>['--ez', 'enable-software-rendering', 'true'],
      if (debuggingOptions.skiaDeterministicRendering) ...<String>['--ez', 'skia-deterministic-rendering', 'true'],
      if (debuggingOptions.traceSkia) ...<String>['--ez', 'trace-skia', 'true'],
      if (debuggingOptions.traceSystrace) ...<String>['--ez', 'trace-systrace', 'true'],
      if (debuggingOptions.dumpSkpOnShaderCompilation) ...<String>['--ez', 'dump-skp-on-shader-compilation', 'true'],
      if (debuggingOptions.debuggingEnabled) ...<String>[
        if (debuggingOptions.buildInfo.isDebug) ...<String>[
          ...<String>['--ez', 'enable-checked-mode', 'true'],
          ...<String>['--ez', 'verify-entry-points', 'true'],
        ],
        if (debuggingOptions.startPaused) ...<String>['--ez', 'start-paused', 'true'],
        if (debuggingOptions.disableServiceAuthCodes) ...<String>['--ez', 'disable-service-auth-codes', 'true'],
        if (debuggingOptions.dartFlags.isNotEmpty) ...<String>['--es', 'dart-flags', debuggingOptions.dartFlags],
        if (debuggingOptions.useTestFonts) ...<String>['--ez', 'use-test-fonts', 'true'],
        if (debuggingOptions.verboseSystemLogs) ...<String>['--ez', 'verbose-logging', 'true'],
      ],
      apk.launchActivity,
    ];
```

You can see a similar formatting (except if one-liner) with `if` sometimes multi-line and sometimes not.

Do you prefer this output with `if` one-liner ?

File: packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart [hunk 2]
```diff
@@ -125,9 +125,7 @@ class MultirootFileBasedAssetReader extends core.FileBasedAssetReader {
   @override
   Stream<AssetId> findAssets(Glob glob, {String package}) async* {
     if (package == null || packageGraph.root.name == package) {
-      final String generatedRoot = fs.path.join(
-        generatedDirectory.path, packageGraph.root.name
-      );
+      final String generatedRoot = fs.path.join(generatedDirectory.path, packageGraph.root.name);
```
Comment (HansMuller) [inline packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart]: NICE

**Review by HansMuller — APPROVED** [review]
LGTM

**Pull request merged**